features/console.feature

```diff
@@ -1,7 +1,45 @@
 Feature: The console command
   Most aspects of connection to a server are tested in shell feature.
 
-  Scenario: Opening a local Rails console
+  Scenario: Opening a local Rails console with an irb version between 1.2 and 1.2.6
+    Given the irb version is "1.2"
     When I run `geordi console`
     Then the output should contain "# Opening a local Rails console"
-      And the output should contain "Util.run! bundle exec rails console -e development"
+      And the output should contain "Util.run! bundle exec rails console -e development -- --nomultiline"
+
+
+  Scenario: Opening a local Rails console with irb version >= 1.2.6
+    Given the irb version is "1.2.6"
+    When I run `geordi console`
+    Then the output should contain "# Opening a local Rails console"
+    And the output should contain "Util.run! bundle exec rails console -e development"
+
+
+  Scenario: Opening a remote Rails console with an irb version between 1.2 and 1.2.6
+    Given the irb version is "1.2"
+    And a file named "Capfile" with "Capfile exists"
+    And a file named "config/deploy.rb" with "deploy file exists"
+    And a file named "config/deploy/staging.rb" with:
+    """
+    set :deploy_to, '/var/www/example.com'
+    set :user, 'user'
+    server 'www.example.com'
+    """
+    When I run `geordi console staging`
+    Then the output should contain "# Opening a Rails console on staging"
+    And the output should contain "Util.run! ssh, user@www.example.com, -t, cd /var/www/example.com/current && bash --login -c 'bundle exec rails console -e  -- --nomultiline"
+
+
+  Scenario: Opening a remote Rails console with irb version >= 1.2.6
+    Given the irb version is "1.2.6"
+    And a file named "Capfile" with "Capfile exists"
+    And a file named "config/deploy.rb" with "deploy file exists"
+    And a file named "config/deploy/staging.rb" with:
+    """
+    set :deploy_to, '/var/www/example.com'
+    set :user, 'user'
+    server 'www.example.com'
+    """
+    When I run `geordi console staging`
+    Then the output should contain "# Opening a Rails console on staging"
+    And the output should contain "Util.run! ssh, user@www.example.com, -t, cd /var/www/example.com/current && bash --login -c 'bundle exec rails console -e"
```

Comment:
This test would be green on v1.2.1 as well – you should add `But the output should not contain "nomultiline"` to all tests that expect the flag to be absent.

Note that negative expectations ("should not") are quite easy to satisfy. E.g. `the output should not contain "-- -nomultiline"` would pass for all three scenarios, by accident. Thus we aim to write them as general as possible.